infrastructure/backups/restore.sh

```diff
@@ -271,3 +282,7 @@ tar -xzvf $ROOT_PATH/backups/vsexport/ocrvs-$LABEL.tar.gz -C $ROOT_PATH/vsexport
 if [ "$IS_LOCAL" = false ]; then
   docker service update --force --update-parallelism 1 opencrvs_migration
 fi
+
+if [ "$RUN_CLEANUP" == "true" ]; then
```

Comment:
Should this be 

```
if [ "$RUN_CLEANUP" == "TRUE" ]; then
```

Is there a benefit using strings and not booleans?

Reply:
Bash doesnt really have booleans so its always a string comparison

Reply:
Ah alright! The `RUN_CLEANUP="TRUE"` above is what caught my attention and just thought if this will ever evaluate to true as in the condition "true" is lowercased